### PR TITLE
Add the ability to not auto-create hot account when activate cold sta…

### PR DESCRIPTION
**Pull request by sondreb**
…king

- Need review and discussion if an additional API endpoint should be added ("setup-offline-staking") or if simply the "SetupColdStakingRequest" type should be expanded with an extra property?

### CODE REVIEW

File: src/Features/Blockcore.Features.ColdStaking/ColdStakingManager.cs [hunk 1]
```diff
@@ -344,7 +346,16 @@ internal HdAddress GetFirstUnusedColdStakingAddress(string walletName, bool isCo
 
             // Get/create the cold staking accounts.
             HdAccount coldAccount = this.GetOrCreateColdStakingAccount(walletName, true, walletPassword);
-            HdAccount hotAccount = this.GetOrCreateColdStakingAccount(walletName, false, walletPassword);
+            HdAccount hotAccount;
+
+            if (createHotAccount)
```
Comment (dangershony) [inline src/Features/Blockcore.Features.ColdStaking/ColdStakingManager.cs:351]: Wait the problem is only with hot accounts?

Comment (sondreb) [reply src/Features/Blockcore.Features.ColdStaking/ColdStakingManager.cs:351]: The problem is having both active at the same time, there is no need for the reversed.

**Comment by dangershony** [comment]
Why all tests failed? I will rerun them

**Comment by dangershony** [comment]
This is now good to push

**Pull request merged**